### DISCUSSION TITLE
How to prevent q-drawer from hiding when clicking on an item inside it ?

### DISCUSSION THREAD

**Discussion by 549393092** [quasarframework/quasar#14574] [General - Components / Directives / etc] [answered]
Dear Quasar team,

There is a silimar topic [10262](https://github.com/quasarframework/quasar/discussions/10262), but seems not answered. So I would like to ask again, could you please have a look?

I reproduce this issue in https://codepen.io/Stanley-549393092/pen/poVOxME?editors=1010.
As you can see, after clicking "three vertial dot" button, it will show additional functions. But at this time, it will hide the drawer.
Can we stop hiding the drawer for this case?

I tried to use @click.stop to prevent button event, but it seems no effect.
This is my requirement. I would like to add addtional function to each menu, for example, add it to favorite or open in new tab.
Not sure if it's the best way to implement it. Appreciate if you could give some proposal.

Regards

**Comment by pdanpdan** [comment, marked as answer]
https://codepen.io/pdanpdan/pen/WNJgPxW?editors=1000

**Reply by 549393092** [reply]
Yes, it's amazing. Thanks a lot!
